spec/models/search/work_search_form_spec.rb

```diff
@@ -145,6 +145,61 @@
       end
     end
 
+    describe "when searching by series title" do
+      let!(:main_series) { create(:series, title: "Persona: Dancing in Starlight", works: [work]) }
+      let!(:spinoff_series) { create(:series, title: "Persona 5", works: [second_work]) }
+
+      it "returns only works in matching series" do
+        run_all_indexing_jobs
+
+        results = WorkSearchForm.new(series_titles: "dancing").search_results
```

Comment:
I would find the spec easier to read if each set of expectations had its own context, to make it clear what aspect of it we're testing, such as "when using a query", "when using a wildcard" etc. It also makes it easier to track where it breaks, if we ever get spec failures.

Reply:
I split the specs, the new output is:
```
WorkSearchForm
  searching
    when searching by series title
      using the "series_titles" field
        returns only works in matching series
        returns only works in matching series with numbers in titles
        returns all works in series for wildcard queries
      using the "query" field
        returns only works in matching series
        returns only works in matching series with numbers in titles
        returns all works in series for wildcard queries
      after a series is renamed
        returns only works in matching series
      after a work is removed from a series
        returns only works in matching series
      after a series is deleted
        returns only works in matching series
```